### ISSUE TITLE
Expo camera is more zoomed in than default iPhone Camera

### ISSUE THREAD

**Issue by applesilver** [expo/expo#5437]
I have implemeneted expo camera compoenet and the camera is more zoomed in than default iPhone Camera (probably 1.1x or 1.2x). I tried to find solution on google but there's no solution for this problem yet on the internet.

Please give me the solution to this

**Comment by rhong7** [comment]
Yes I've tried that. And I have tried both iphone 7 and iphone xr.

**Comment by Flow11** [comment]
I have tried on expo 35, on an Ipad mini 4 and I also reproduce the issue.
Any update on this ?
Other related issues: #2967 #1527

**Comment by ksairi** [comment]
@ide do we know when this could be fix? sdk 36?

**Comment by impactcolor** [comment]
On looking into this on my iphone it selects the zoom of the video recording camera not the picture photo camera view.  From the expo docs it appears there's no difference between CAMERA for photo camera and video recording camera.

**Comment by brentvatne** [comment]
react-native-camera (which is based on expo-camera) attempted to handle this adding a `maxZoom` prop: https://github.com/react-native-community/react-native-camera/pull/2510

we could maybe do something similar

**Comment by sungsong88** [comment]
I have two projects: one with SDK35 and the other one with SDK37, and both still have this problem. Any fixes yet?

**Comment by davidcian** [comment]
Any news on this? I have an app which relies on the camera, and while I'm down-playing the issue for the time being, it would be nice to have the same zoom as the default iOS camera app.

**Comment by smirea** [comment]
any updates on this?

**Comment by brentvatne** [comment]
@smirea - this is the only update:

> react-native-camera (which is based on expo-camera) attempted to handle this adding a `maxZoom` prop: [react-native-camera/react-native-camera#2510](https://github.com/react-native-camera/react-native-camera/pull/2510). lots of discussion in [react-native-camera/react-native-camera#1772](https://github.com/react-native-camera/react-native-camera/issues/1772)
> 
> we could maybe do something similar. open to pull requests!

**Comment by aimenBrahmi** [comment]
Expo V 41.0.1 same issue.

**Comment by umutsesen** [comment]
still same.. really cannot believe it

**Comment by smirea** [comment]
still happening

**Comment by villeliekari** [comment]
Adding `pictureSize={"Photo"}` inside Camera component worked for me. [https://docs.expo.dev/versions/latest/sdk/camera/#getavailablepicturesizesasync](https://docs.expo.dev/versions/latest/sdk/camera/#getavailablepicturesizesasync)

**Comment by jaltin** [comment]
> Adding `pictureSize={"Photo"}` prop inside Camera component worked for me. https://docs.expo.dev/versions/latest/sdk/camera/#getavailablepicturesizesasync

I tried this but unfortunately it doesn't work for me (IPhone XS).

**Comment by jaltin** [comment]
Still an issue

**Comment by brentvatne** [comment]
> react-native-camera (which is based on expo-camera) attempted to handle this adding a `maxZoom` prop: [react-native-camera/react-native-camera#2510](https://github.com/react-native-camera/react-native-camera/pull/2510). lots of discussion in [react-native-camera/react-native-camera#1772](https://github.com/react-native-camera/react-native-camera/issues/1772)
> 
> we could maybe do something similar. open to pull requests!

not much can be added on top of this. if you'd like to open a pr to fix it, it's welcome. i will lock the issue now and we can discuss further in a pr
